packages/www/src/pages/index.tsx

```diff
@@ -210,9 +207,9 @@ export default function Homepage() {
           <div class="info">
             <h2>📱 Multi-platform</h2>
             <p>
-              Brisa allows you to build web applications that can be easily
-              converted to native applications for <b>Android</b>, <b>iOS</b>,
-              and <b>desktop</b>. It's full integrated with Tauri.
+              Brisa can also be used with Tauri, which allows you to build web
```

Comment:
Brisa implements Tauri. On this one I like more the original sentence.

Reply:
I might be missing something here, then. Because as far as I know, you use and compile React, JavaScript, or Angular with Electron or Tauri. I wouldn't say they “implement” them, unless there is some sort of API support, in which case it's more of an integration. I would love to better understand how Brisa works with Tauri, @aralroca can you help? 
The way I see it, if Brisa can be compiled to be embedded with Tauri because in the end it's just JS, then it can be used with Tauri.

Reply:
Tauri is almost transparent for the developer in Brisa, this means that only needs to modify one configuration property in `brisa.config.ts` file to decide if build an .apk, .ica, .exe, .dmg, etc. In this way facilitate transforming your web app to an mobile/desktop app in a easy way. So, our CLI configures and integrates Tauri to do this, but is a Brisa feature.

Reply:
@gariasf after this change we can merge the PR